packages/validation-manager/src/ERC7562Parser.ts

```diff
@@ -802,4 +834,17 @@ export class ERC7562Parser {
       }
     }
   }
+
+  private _isPrecompiled (address: string): boolean {
```

Comment:
bad test: instead of checking on the address, check if it has code: a precompiled contract is a call to an address without code (e.g. any L2 has its own precompiles, obviously are all forbidden)

Reply:
Addresses without code are forbidden by OP-041 anyways. This is a double-check I think, in case some "precompile" actually has a solidity code injected into it, right?

Reply:
right.. we're only showing example precompiles.. 
also, the separation of isPrecompile and forbidden is not really required... we're only interested in allowedPrecompiles. anything else (without code) is forbidden